Pipfile

```diff
@@ -8,6 +8,8 @@ python_version = "3.10"
 
 [dev-packages]
 pytest = "*"
+pyarrow = "*"
 pre-commit = "*"
 sceptre = ">=3.2.0"
 sceptre-sam-handler = "*"
+synapseclient = "==2.7.*"
```

Comment:
Nit: You may want to lock down the dependencies here. 

Instead of doing `==2.7.*` and can do `~=2.7`.  See this [slackoverflow](https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean)